src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue1439.cs

```diff
@@ -12,30 +12,29 @@ public Issue1439(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "ItemTapped event for a grouped ListView is not working as expected.";
 
-	// [Test]
-	// [Category(UITestCategories.TableView)]
-	// [FailsOnIOS]
-	// public void Issue1439Test()
-	// {
-	// 	App.WaitForElement(q => q.Marked(A));
-	// 	App.Tap(q => q.Marked(A));
+	[Test]
+	[Category(UITestCategories.TableView)]
+	public void Issue1439Test()
+	{
+		App.WaitForElement("A");
+		App.Tap("A");
 
-	// 	Assert.AreEqual(A, App.WaitForElement(q => q.Marked(lblItem))[0].ReadText());
-	// 	Assert.AreEqual(Group_1, App.WaitForElement(q => q.Marked(lblGroup))[0].ReadText());
+		Assert.That(App.FindElement("lblItem").GetText(), Is.EqualTo("A"));
```

Comment:
Well adapted. Just in case, if want to leave the same Asserts, need to use `ClassicAssert`.

Example:

`ClassicAssert.AreEqual`